### PR TITLE
v1.8.0 - colour updates, accessibility pages added and bug fixes

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Documentation page on browser support added
- Documentation page on accessibility checklist added

### Changed
- Updated Colour Palette in line with new brand changes
- Updated Accessibility page links
- Comments updated to be consistent throughout SCSS files

### Fixed
- Site nav fixed on mobile devices

### Removed
- Old IE CSS file - not needed in documentation as not supporting <IE9

### CODE REVIEW

File: docs/src/templates/pages/documentation/css/index.hbs [hunk 1]
```diff
@@ -86,66 +86,41 @@ Colours specific to components should be defined at the top of each component sc
 
 As is considered best practice, we develop with a mobile-first approach to our projects.
 
-Media queries are typically handled with a [set of useful mixins](https://github.je-labs.com/I18N-Production-ROTW/ConsumerWeb/tree/master/src/ConsumerWeb/Assets/src/scss/mixins/_responsive.scss). These are used so that we can define separate media-query content for `< IE9` and browsers with support for media queries. These are directly related to the global `$fix-mqs` var defined in `js-old-ie.scss`. `js-old-ie.scss` ignores any content in media-queries with values less than the `$fix-mqs` var value.
+On the global platform, we recommend using the [@include-media](https://include-media.com/) Sass library. This provides an elegant way to define our media queries in a more readbale way while also hooking into  the [breakpoint map](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L65-L72) defined in fozzie's variables.
```
[the remark this answers is not in the document]
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/css/index.hbs]: Good idea – didn't know you could do that 👍

File: docs/src/templates/pages/documentation/general/browser-support.hbs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+section-title: General Documentation
+description: The platforms and browsers that we support
+docs: true
+
+navgroup: documentation
+navsub: docs-general
+navactive: browser-support
+---
+
+The number of browsers available to our users is vast, especially for those using mobile devices.  Below, we have defined a rating system that helps us to specify the level of browser support we aim to achieve and to clarify what level of testing is required.
```
[the remark this answers is not in the document]
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/general/browser-support.hbs]: lol – I have no idea why this is engrained in me to do this…I just can't stop double spacing whenever I write docs.

Out of interest this apparently comes from monospaced fonts were double spacing was used to emphasise spacing at the end of paragraphs (like on a typewriter).  No idea why I do it though…!

File: docs/src/templates/pages/documentation/css/index.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/css/index.hbs]: Fixed

File: docs/src/templates/pages/documentation/css/index.hbs [hunk 3]
```diff
@@ -86,66 +86,41 @@ Colours specific to components should be defined at the top of each component sc
...
 
 #### Media query example
 
 ```scss
 /**
- * This Sass mixin:
- * The value for the width should be unitless and we suggest
- * using your predefined breakpoint variables as the argument
+ * Using @include-media you can specify breakpoints
+ * using the map key's defined in Fozzie's [breakpoint map](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L65-L72)
```
[the remark this answers is not in the document]
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/css/index.hbs]: Fixed

File: docs/src/templates/pages/documentation/general/browser-support.hbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/general/browser-support.hbs]: Fixed

[the remark this answers is not in the document]
Comment (DamianMullins) [reply docs/src/templates/pages/documentation/general/browser-support.hbs]: I didn't know that was even a thing!

**Comment by ashleynolan** [comment]
Merging this with one review as it's only docs updates (and all of the other guys are at the cinema this afternoon 😉)

**Comment by DamianMullins** [comment]
🤠

File: docs/src/templates/pages/documentation/general/browser-support.hbs [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+<table class="o-table c-matrix not-striped" role="grid">
+    <thead>
+        <tr role="row">
+            <th data-column="0" tabindex="0" scope="col" role="columnheader" aria-disabled="false" unselectable="on" aria-sort="none">Browser</th>
+            <th data-column="1" tabindex="0" scope="col" role="columnheader" aria-disabled="false" unselectable="on" aria-sort="none">Grading</th>
+        </tr>
+    </thead>
+    <tbody aria-live="polite" aria-relevant="all">
+        <tr role="row">
+            <td>Chrome 59+ (MacOS &amp; Windows)</td>
```
Comment (DamianMullins) [inline docs/src/templates/pages/documentation/general/browser-support.hbs]: Have just noticed that some of these browser versions are quite outdated - Chrome is currently on v66) - are these versions taken from current analytics? Or should we update them?

Comment (ashleynolan) [reply docs/src/templates/pages/documentation/general/browser-support.hbs]: They're taken from the analytics.  These should get reviewed and amended every 6 months or so.

**Pull request merged**